core/src/test/java/org/apache/carbondata/core/util/CarbonUtilTest.java

```diff
@@ -928,6 +928,11 @@ public void testTupeIDInUpdateScenarios() {
     Assert.assertEquals(CarbonTablePath.getShortBlockId(blockId), "0/0-0_0-0-0-1597409791503");
     blockId = "c3=aa/part-0-100100000100001_batchno0-0-0-1597411003332.snappy.carbondata";
     Assert.assertEquals(CarbonTablePath.getShortBlockIdForPartitionTable(blockId), "c3=aa/0-100100000100001_0-0-0-1597411003332");
+    // CACHE_LEVEL = BLOCKLET case
```

Comment:
Please add a functional testcase for this scenario and check data after setting cache_level=blocklet

Reply:
Done